docs/getting-started/pipeline.yaml

```diff
@@ -22,11 +21,6 @@ spec:
     - name: build-docker-image
       taskRef:
         name: build-docker-image
-      conditions:
-      - conditionRef: "go-test"
-        resources:
-          - name: source-repo
-            resource: source-repo
```

Comment:
Can we add replacement of conditions ?

I mean conditions are doing `go test` if we remove it now how do we run `go test` :thinking:

Reply:
I removed it as it's not important. We don't need that much focus on pipelines.

Reply:
Okay 

If require may be we can update/add later

I will merge this PR to unblock CI failure :+1: